### PR TITLE
History management

### PR DESCRIPTION
**Pull request by maitrakhatri**
# features
- when user open a video, it gets added to history
- user can delete one specific video from the history
- user can clear the history at once

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]
Good work! You can merge the changes

File: src/Context/history-context.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    const getHistory = async () => {
+        try {
+            const response = await axios.get('/api/user/history', {
+                headers: {
+                    authorization: token
+                }
+            })
+            console.log(response.data.history)
+            setMyHistory(response.data.history)
```
Comment (Sharmaryan) [inline src/Context/history-context.js:18]: remove console log

**Review by Sharmaryan — APPROVED** [review]
Everything is fine, just remove console logs, I have seen in many places

**Comment by maitrakhatri** [comment]
> Everything is fine, just remove console logs, I have seen in many places

done

**Pull request merged**